### DISCUSSION TITLE
Error Parsing Type Definition with orm.SoftDeletes

### DISCUSSION THREAD

**Discussion by xx1196** [goravel/goravel#442] [Q&A]
I am encountering an issue with Swagger in Go where it fails to parse the type definition orm.SoftDeletes in my model. The specific error messages are:
`2024/06/22 22:14:19 Error parsing type definition 'models.User': : cannot find type definition: orm.SoftDeletes
2024/06/22 22:14:19 ParseComment error in file \app\http\controllers\user_controller.go :models.Municipality: : cannot find type definition: orm.SoftDeletes
`

Model
Here is the model where the issue occurs:

`package models

import (
	"github.com/goravel/framework/database/orm"
)

type User struct {
	Code        string `json:"code"`
	Name        string `json:"name"`
	Description string `json:"description"`
	orm.SoftDeletes
}
`

Controller
Here is the controller where the issue occurs:

`
// @Success 200 {object} models.Municipality
`

**Comment by hwbrzzl** [comment]
Emm, it seems to be a swagger issue, I also want to know the answer, please google it and let me know. Thanks!

**Reply by xx1196** [reply]
The solution is not to use orm.softdeletes or timestamps, but this makes it impossible to do logical deletions
